README.md

```diff
@@ -71,3 +71,47 @@ cl login <host:grpcPort>
 ### Command Completion
 
 The CLI offers command completion for most shells using the `cl completion` command. Specific instructions to install the shell completions can be accessed using `cl completion --help`.
+
+## Common Issues
+
+<details>
+<summary>Could not find protoc plugin for...</summary>
+
+#### Error Message
+Failure: plugin grpc-gateway: could not find protoc plugin for name grpc-gateway -<br>
+please make sure protoc-gen-grpc-gateway is installed and present on your $PATH
+
+#### Solution
+
+```bash
+go get google.golang.org/grpc/cmd/protoc-gen-go-grpc
```

Comment:
The idea about common issues is a good one, I am just not too sure about the particular problems. Aren't these steps all detailed in https://github.com/clouditor/clouditor/blob/main/README.md#build ? Maybe the commands in the build section need to be updated. At least we should not re-repeat the steps again but maybe just put a link to the build section?